src/test/java/seedu/address/logic/parser/AnakinEditDeckParserTest.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.INVALID_DECK_NAME_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_DECK_NAME_A_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_DECK_NAME_B_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_NAME_DECK_A;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_NAME_DECK_B;
+import static seedu.address.logic.parser.AnakinCommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.AnakinCommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.AnakinTypicalIndexes.INDEX_FIRST_DECK;
+
+import org.junit.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.anakincommands.AnakinEditDeckCommand;
+import seedu.address.logic.anakincommands.AnakinEditDeckCommand.EditDeckDescriptor;
+import seedu.address.logic.anakinparser.AnakinEditDeckCommandParser;
+import seedu.address.model.person.Name;
```

Comment:
This might cause problems if latter we want to delete addressbook's code

Reply:
which part? The model.person.Name?